core/chains/evm/txmgr/eth_broadcaster.go

```diff
@@ -118,14 +119,14 @@ type EthBroadcaster struct {
 }
 
 // NewEthBroadcaster returns a new concrete EthBroadcaster
-func NewEthBroadcaster(orm ORM, ethClient evmclient.Client, config Config, keystore KeyStore,
+func NewEthBroadcaster[HEAD any](orm ORM, ethClient evmclient.Client, config Config, keystore KeyStore,
```

Comment:
on EthBroadcaster: it seems to me there should be `Broadcaster[HEAD any]` generic struct somewhere and EthBrodcaster extends it with eth specific stuff.

Reply:
Yes, you are right.
It will be done as part of this ticket: https://smartcontract-it.atlassian.net/browse/BCI-863